src/app/(route)/live/[room_id]/components/LiveStreaming/LiveDetails/HashTag.client.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from 'next/link'
+import React from 'react'
+
+interface HashTagLiProps {
+    tagname:string
+    path:string
+}
+
+function LiveHashTag({ tags }: { tags: HashTagLiProps[] }) {
+    
+  return (
+    <ul id="관련태그" className="text-[#666] mt-[8px] overflow-hidden flex flex-wrap text-[11px] gap-[4px] leading-[19px]">
+        {tags?.length > 0 &&
+            tags.map((tag, idx) => (
+            <HashTagLi key={`${tag.tagname}-${idx}`} {...tag} />
+        ))}
```

Comment:
tags가 빈배열일때 아무것도 렌더링 하지 않는다는걸 의도하신걸까요?!?!
그렇다면 아래 코드가 더 깔끔해보이는데 기준님 생각은 어떠세요?!

```suggestion
        {tags.map((tag, idx) => (
          <HashTagLi key={`${tag.tagname}-${idx}`} {...tag} />
        ))}
```

Reply:
배열로 받을 때 혹시 태그가 없을 수 있어서 위처럼 코드를 짰어요
만약에 태그가 무조건 존재한다면 아래 코드로 수정하겠습니당

Reply:
태그가 없더라도 어차피 빈배열이기때문에 아무것도 렌더링되지 않을것같습니다!